python/ray/_private/runtime_env.py

```diff
@@ -213,13 +213,17 @@ def get_project_package_name(working_dir: str, py_modules: List[str],
     hash_val = None
     excludes = {Path(p).absolute() for p in excludes}
     if working_dir:
-        assert isinstance(working_dir, str)
-        assert Path(working_dir).exists()
+        assert isinstance(working_dir, str), f"`working_dir` is not a string."
```

Comment:
Please don't use `assert` statements for user-facing errors. Best practice here is to use `TypeError` for a type check and `ValueError` for a value check (e.g., an int must be >= 0 or the directory must exist)